threatfox/package.yaml

```diff
@@ -0,0 +1,151 @@
+id: threatfox
+name: ThreatFox
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: https://raw.githubusercontent.com/tenzir/library/main/threatfox/package.svg
+
+description: |
+  [ThreatFox](https://threatfox.abuse.ch/) is a free platform from abuse.ch with
+  the goal of sharing indicators of compromise (IOCs) associated with malware
+  with the infosec community, AV vendors and threat intelligence providers.
+
+  This package provides multiple contexts for enrichment, each of which contains
+  a subset of the ThreatFox IOC database for a particular IOC type and a
+  corresponding pipeline that periodically refreshes the context. The context
+  types are:
+
+  - **(IP, port) tuples**: a record key of type `<ip: ip, port: uint64>`, e.g.,
+    `{ip: "1.2.3.4", port: 80}`.
+  - **Domains**: a string key with domains, e.g., `evil.com`.
+  - **URLs**: a string key of URLs with scheme, e.g.,
+    `https://evil.com/malware.exe`
+  - **Hashes**: a string key with the hashes in the database, i.e., MD5, SHA1,
+    and SHA256. For example, `2d29a1c2e5c6d26bc103343637562205`.
+
+inputs:
+  days:
+    name: Number of days
+    description: |
+      Number of days to filter IOCs for (min: 1, max: 7)
+    default: 3
+
+  refresh-interval:
+    name: Time between context updates
+    description: |
+      The duration between context updates that fetch the ThreatFox database via
+      the API.
+    default: 1 hour
+
+contexts:
+  threatfox_ipports:
+    type: lookup-table
+    description: The subset of the ThreatFox IOC Database for (IP, port) tuples.
+
+  threatfox_domains:
+    type: lookup-table
+    description: The subset of the ThreatFox IOC Database for domains.
+
+  threatfox_urls:
+    type: lookup-table
+    description: The subset of the ThreatFox IOC Database for URLs.
+
+  threatfox_hashes:
+    type: lookup-table
+    description: The subset of the ThreatFox IOC Database for hashes.
```

Comment:
I think these should mention the format of the indicators in the description. Especially for the ip and port pairs this would help.

Dashes or underscores for context names?

Reply:
In any case, I addressed this.